pkg/cryptoutil/verifier.go

```diff
@@ -97,3 +99,33 @@ func NewVerifierFromReader(r io.Reader, opts ...VerifierOption) (Verifier, error
 
 	return NewVerifier(key, opts...)
 }
+
+func NewCAVerifier(certs []*x509.Certificate, opts ...VerifierOption) Verifier {
```

Comment:
This should be unnecessary with the existing X509Verifier, also I see no usages of this